crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -205,10 +206,14 @@ impl ParsedQObject {
                 ImplItem::Method(method) => {
                     self.parse_impl_method(method)?;
                 }
+                ImplItem::Const(constant) => {
+                    self.passthrough_impl_items
+                        .push(ImplItem::Const(constant.clone()));
+                }
                 _ => {
```

Comment:
what about `ImplItem::Type`, `ImplType::Macro`, and `ImplType::Verbatim`, some or all of these seems relevant to pass through as well ? Eg there isn't much difference between Const and Type ?

Reply:
Good point :+1: 
Now passed through as well